bindings/matlab/+iDynTreeWrappers/generalizedBiasForces.m

```diff
@@ -0,0 +1,36 @@
+function h = generalizedBiasForces(KinDynModel)
+
+    % GENERALIZEDBIASFORCES retrieves the generalized bias forces from 
+    %                            the reduced model. 
+    %
+    % This matlab function wraps a functionality of the iDyntree library.                     
+    % For further info see also: http://wiki.icub.org/codyco/dox/html/idyntree/html/
+    %
+    % FORMAT:  h = generalizedBiasForces(KinDynModel)
+    %
+    % INPUTS:  - KinDynModel: a structure containing the loaded model and additional info.
+    %
+    % OUTPUTS: - h: [ndof+6 x 1] generalized bias accelerations.
```

Comment:
Where it is documented if we are using (base, joint) serialization and not (joint, base)? Explicitly to avoid this confusion,  I suggest to indicate the vectors as `6+nDof` instead of `nDof+6`. 
```suggestion
    % OUTPUTS: - h: [6+nDof x 1] generalized bias accelerations.
```